distributor/cmd/main.go

```diff
@@ -83,9 +92,9 @@ func _main(args []string, env envConfig) int {
 	createEventForwardingEndpoint(env)
 
 	// initialize the http client
-	connectionType := strings.ToLower(os.Getenv("CONNECTION_TYPE"))
+	//connectionType := strings.ToLower(os.Getenv("CONNECTION_TYPE"))
```

Comment:
the check for the connection type should not be removed, since we would like to have the possibility to subscribe to events either via NATS or via HTTP polling